packages/plugin-char/src/Char.ts

```diff
@@ -1,5 +1,6 @@
 import { JWPlugin, JWPluginConfig } from '../../core/src/JWPlugin';
 import { CharNode } from './CharNode';
+import { LineBreakNode } from '../../plugin-linebreak/src/LineBreakNode';
```

Comment:
Be careful when referencing plugins in other plugins. This one should not be mandatory for `Char` to work.